package-lock.json

```diff
@@ -17382,7 +17382,6 @@
     "node_modules/mmjstool": {
       "version": "1.0.0",
       "resolved": "git+ssh://git@github.com/mattermost/mattermost-utilities.git#3faa6075089a541d8c90ed85114e644c7a23fedf",
-      "integrity": "sha512-84xbA6yRt+pEbYk/OmM9sbJQp7PkSAgqvn0N/QXuj1RKR+upxdAa6vTbTTBA+XweECN6xebCWncGJJiqWrtR1g==",
```

Comment:
I can revert them to original, not sure what the best approach is since its a [known issue](https://github.com/npm/cli/issues/2846) but removing them entirely may just work.